aitelemetry/telemetrywrapper.go

```diff
@@ -116,9 +116,12 @@ func getMetadata(th *telemetryHandle) {
 		debugLog("[AppInsights] Error initializing kvs store: %v", err)
 		return
 	}
-
-	err = kvs.Lock(store.DefaultLockTimeout)
-	if err != nil {
+	// Acquire store lock. For windows 1m timeout is used while for Linux 10s timeout is assigned.
+	var lockTimeoutValue time.Duration = store.DefaultLockTimeout
```

Comment:
l dont think we should change for telemetry service.. lets keep the same

Reply:
I recall when I left the telemetry timeout unchanged the result got weird!